### ISSUE TITLE
Can’t import key from clipboard

### ISSUE THREAD

**Issue by nathanielwheeler** [blinksh/blink#717]
I can’t copy a key into the config file from the clipboard.  I’ve restarted and reinstalled, but all I get is “Invalid Key: clipboard content couldn’t be validated as a key.”
The key in question is a normal rsa public key generated on the linux machine I’m trying to connect to.

**Comment by yury** [comment]
Hi @ObliviNate,

Yep, it is known issue. Since new openssh versions start to store keys in new format.

You should add -m PEM to generate key in pem format that blink understands.

`ssh-keygen -m PEM -f path/to/key`

So it starts with
```
-----BEGIN RSA PRIVATE KEY-----
```
and not 
```
-----BEGIN OPENSSH PRIVATE KEY-----
```

Sorry for inconvenience, we have solution for that issue, but really it is more like hack, so we don't push it yet.

**Comment by ObliviNate** [comment]
That’s how the beginning of my rsa file is.
‘’
-----BEGIN RSA PRIVATE KEY----- Proc-Type: 4,ENCRYPTED DEK-Info: AES-128-CBC, <etc>
‘’
I am flabbergasted.  I’ve been going through openssh documentation and everything, can’t figure out what it is

**Comment by yury** [comment]
Hi @ObliviNate,

It looks ok for me. I will try to reproduce and fix that. But for now, you can leave passphrase blank, so no encryption is performed for your key (not very safe, but as workaround)

**Comment by yury** [comment]
Fixed in 12.9. 
Closing this issue. Feel free to reopen.
